internal/cmd/artifacts/upload.go

```diff
@@ -18,7 +18,7 @@ func UploadCommand() *cobra.Command {
 	var out string
 
 	cmd := &cobra.Command{
-		Use:   "upload",
+		Use:   "upload <jobID> <filename>",
 		Short: "Uploads an artifacts for the job.",
 		Long:  "Uploads an artifacts for the job. Real Device job is not supported.",
```

Comment:
Figure may as well fix the grammar while you're improving the messaging.

```suggestion
		Short: "Uploads an artifact for the job.",
		Long:  "Uploads an artifact for the job. Real Device job is not supported.",
```